docs/book/src/topics/addons.md

```diff
@@ -0,0 +1,162 @@
+# CNI
+
+By default, the CNI plugin is not installed for self-managed clusters, so you have to [install your own](https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution).
+
+Some of the instructions below use [Helm](https://helm.sh) to install the addons. If you're not familiar with using Helm to manage Kubernetes applications as packages, there's lots of good [Helm documentation on the official website](https://helm.sh/docs/). You can install Helm by following the [official instructions](https://helm.sh/docs/intro/install/).
+
+## Calico
+
+To install [Calico](https://www.tigera.io/project-calico/) on a self-managed cluster using the office Calico Helm chart, run the commands corresponding to the cluster network configuration:
+
+### For IPv4 clusters
+
+```bash
+IPV4_CIDR_BLOCK=<cluster ipv4 pod cidr block> \
+helm repo add projectcalico https://projectcalico.docs.tigera.io/charts && \
+helm install calico projectcalico/tigera-operator -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico/values.yaml --set-string installation.calicoNetwork.ipPools[0].cidr="$IPV4_CIDR_BLOCK" --namespace tigera-operator --create-namespace
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico/felix-override.yaml
+```
+```
+
+### For IPv6 clusters
+
+```bash
+IPV6_CIDR_BLOCK=<cluster ipv6 pod cidr block> \
+helm repo add projectcalico https://projectcalico.docs.tigera.io/charts && \
+helm install calico projectcalico/tigera-operator -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico-ipv6/values.yaml  --set-string installation.calicoNetwork.ipPools[0].cidr="$IPV6_CIDR_BLOCK" --namespace tigera-operator --create-namespace
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/addons/calico/felix-override.yaml
+```
+
+### For IPv4/IPv6 dual-stack clusters
```

Comment:
I think I'd prefer referring to this as something like "dual-stack (IPv4 + IPv6)"